budgetportal/migrations/0044_auto_20200305_0938.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 2.2.10 on 2020-03-05 09:38
+
+import autoslug.fields
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("budgetportal", "0043_auto_20200304_1335"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="infrastructureprojectpart",
```

Comment:
we don't want to be renaming this - it's also national - please remove and I'll roll back this one on my side

Reply:
This migration rolls it back. Going to squash those migrations once tests pass.

Reply:
by squash I mean remove those 2 renamings

Reply:
hah - sorry, I was also tired :)